app/assets/stylesheets/app/consent_form_preview.scss

```diff
@@ -0,0 +1,38 @@
+.consent-agreement {
```

Comment:
I agree that a component based approach is the right way to go. A concern however is that when you (prematurely) define a component that hasn't been through the rigour of a design process and proven as being a truly reusable pattern, you still end up using this component for anything that approximately looks like a fit. 

If there is a way to _revisit the components, and revise their design_ that has a lower overhead than proving that they truly are universal components up front (through a design process over several use cases) then we don't have a problem with defining components in the first use case.

$0.02